docs/docs/tutorials/classification_finetuning/index.ipynb

```diff
@@ -12,16 +12,25 @@
     "\n",
     "### Install dependencies and download data\n",
     "\n",
-    "Install the latest DSPy via `pip install -U --pre dspy` and follow along. This tutorial depends on DSPy 2.6.0 (pre-release).\n",
+    "Install the latest DSPy via `pip install -U --pre dspy` and follow along (or `uv pip`, if you prefer). This tutorial depends on DSPy 2.6.0 (pre-release).\n",
```

Comment:
"This tutorial depends on DSPy 2.6.0 (pre-release)" reads like it only works with 2.6.0, shall we just require dspy>=2.6.0 by changing the command to `pip install -U dspy>=2.6.0`?